include/envoy/buffer/buffer.h

```diff
@@ -77,6 +79,19 @@ class Instance {
    */
   virtual void add(const Instance& data) PURE;
 
+  /**
+   * Prepend a string to the buffer.
+   * @param data supplies the string to copy.
```

Comment:
Hopefully not too picky but string->string_view on both lines?

Reply:
Updated